### PR TITLE
Duration seconds

### PR DESCRIPTION
**Pull request by dfish3r**

### CODE REVIEW

File: core/src/main/java/org/ldaptive/cache/LRUCache.java [hunk 1]
```diff
@@ -75,7 +75,7 @@ protected boolean removeEldestEntry(final Map.Entry<Q, Item> entry)
         }
       }
     };
-    executor.scheduleAtFixedRate(expire, interval.getSeconds(), interval.getSeconds(), TimeUnit.SECONDS);
+    executor.scheduleAtFixedRate(expire, interval.toMillis(), interval.toMillis(), TimeUnit.MILLISECONDS);
```
Comment (serac) [inline core/src/main/java/org/ldaptive/cache/LRUCache.java]: Why this change? So you can have a Duration down to ms resolution?

Comment (dfish3r) [inline core/src/main/java/org/ldaptive/cache/LRUCache.java]: Yes, but mostly for consistency.
